### ISSUE TITLE
Hope to add quick Settings Tile to Android models

### ISSUE THREAD

**Issue by NianBroken** [localsend/localsend#310]
### Is your feature request related to a problem? Please describe.

Hope to add quick Settings Tile to Android models

### Describe the solution you'd like

![Screenshot_2023-03-22-21-56-07-563_mark via](https://user-images.githubusercontent.com/57057288/226926784-69735967-062f-4dce-8fa0-79c2eb098f70.jpg)

**Comment by sergd88** [comment]
I agree! This is a quite wonderful and lovely feature that adds convenience and maneuverability. I hope to see it later.

**Comment by cxfksword** [comment]
Wish also has a tile to quickly send and edit clipboard text

**Comment by Anonuser-01** [comment]
+1
